library/src/main/java/com/novoda/downloadmanager/Migration.java

```diff
@@ -21,6 +21,15 @@ Batch batch() {
         return fileMetadata;
     }
 
+    boolean hasDownloadedBatch() {
```

Comment:
Might be able to switch this with a better query that checks the batch size from the v1 database. I'd rather merge this and make improvements in a follow-up. We know this works after all 😄